README.md

```diff
@@ -8,6 +8,46 @@ This DOES NOT install a Chef client on your Jenkins server, that needs to be don
 The minimum Jenkins version we're building for is LTS 1.554.3.
 Get your bearings with <https://wiki.jenkins-ci.org/display/JENKINS/Plugin+tutorial>
 
+## Install plugin
+
+**Step 1**: Build .hpi file
+
+```
+mvn clean -DskipTest package
+```
+
+**Step 2**: In jenkins server
+
+- Go to **Manage Jenkins** -> **Manage Plugins** -> **Advanced** -> **Upload Plugin**
+
+- Select file chef-identity.hpi and upload to jenkins server
+
+## Chef Identity Setting
+
+Go to **Manage Jenkins** -> **Configure System** -> **Chef Identity Management**
+
+- Identity Name
+
+- user.pem key: Key pair download from AWS
```

Comment:
Let's change AWS here to "Chef Server" because it is not always a true statement that the key pair is downloaded from AWS.